ci-operator/config/openshift/release/openshift-release-master__nightly-4.21.yaml

```diff
@@ -525,6 +525,103 @@ tests:
   steps:
     cluster_profile: equinix-edge-enablement
     workflow: baremetalds-two-node-fencing
+- as: e2e-metal-ovn-two-node-fencing-degraded-techpreview
+  cluster: build05
+  cron: '@daily'
```

Comment:
With metal jobs, we tried not to use daily as the cron expression. You can use [this dashboard ](https://ci-route-ci-grafana.apps.ci.l2s4.p1.openshiftapps.com/d/628a36ebd9ef30d67e28576a5d5201fd/boskos-dashboard?from=now-1d&to=now&timezone=browser) to find a time of the day where metal platform is light to start this job.